lib/stripe.rb

```diff
@@ -77,6 +77,7 @@
 require "stripe/recipient_transfer"
 require "stripe/refund"
 require "stripe/reversal"
+require "stripe/sigma/scheduled_query_run"
```

Comment:
Kind of nice/lucky that this wasn't brought in until you'd established the new namespacing convention.

Reply:
full disclosure, I saw that resource once and tried on a week end, and failed and thought "well no one has asked for it so it's okay" 😂

Reply:
lol! Well, pretty nice that it's a piece of cake nowadays right?